### ISSUE TITLE
Create a PR template to standardize its format

### ISSUE THREAD

**Issue by liutaurasvilda** [CoderanchCorral/Blackjack#31]
Create PR template to standardize its format.

Template could include:
- Description
- Testing Methodology used
- Potential problems / open questions
- PR Checklist ?

Discuss with the team if we want and how we want. 

For consideration as well:
- PR title standardization
- Branch naming convention, i.e. `[feature | refactor | bugfix | doc]/issue-no/descriptive-name-of-change`

**Comment by nibsi** [comment]
Agreed. We should also update the `CONTRIBUTING` page to explain a bit better how to write new issues and how to create PRs.

Could you take the lead in this one by making a proposal?

**Comment by liutaurasvilda** [comment]
Yes, happy to do so, I'd just move at a bit slower pace due to some other commitments.

**Comment by nibsi** [comment]
No problem, we're here to enjoy the project, not to get stressed out by it.
